clikit/src/main/java/org/adamalang/clikit/model/Argument.java

```diff
@@ -0,0 +1,54 @@
+package org.adamalang.clikit.model;
+
+import org.adamalang.clikit.exceptions.XMLFormatException;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+public class Argument {
+    /** Represents an argument that is used by a command **/
```

Comment:
move to prior the "public class" as that will associate the comment with the class in javadoc